src/App/docutils/ComponentPreview/index.js

```diff
@@ -381,7 +385,7 @@ const ComponentPreview = ({ children, language, removeOuterTag, hideValue, hideC
 };
 
 ComponentPreview.propTypes = {
-    language: PropTypes.oneOf(["html", "javascript", "css", "terminal"]).isRequired,
+    language: PropTypes.oneOf(["html", "javascript", "css", "terminal", "json"]).isRequired,
```

Comment:
Do we need json somewhere? 🤔

Reply:
No, but it is marked as an option further up. 
I added it because of this warning: 
![image](https://user-images.githubusercontent.com/21359433/153571038-9b859a3c-a8f1-4a40-a046-755c64ea609b.png)